### PR TITLE
Add more Scala versions to crossScalaVersions

### PR DESCRIPTION
**Pull request by azolotko**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -8,7 +8,7 @@ val slf4jApi = "org.slf4j" % "slf4j-api" % slf4jV
 val slf4jSimple = "org.slf4j" % "slf4j-simple" % slf4jV
 val specs2 = "org.specs2" %% "specs2-core" % specs2V % "it,test"
 val specs2Mock = "org.specs2" %% "specs2-mock" % specs2V % "test"
-val enumUtils = "com.thenewmotion" %% "enum-utils" % "0.2.1"
+val enumUtils = "com.thenewmotion" %% "enum-utils" % "0.3-SNAPSHOT"
```
Comment (sbmpost) [inline build.sbt]: Reminder to remove SNAPSHOT

**Comment by sbmpost** [comment]
Build is failing for some reason

**Comment by azolotko** [comment]
@sbmpost Apparently, we have two `enum-utils` repos and they differ in certain ways.

**Comment by reinierl** [comment]
Hmm, what happened to coverage?

**Comment by reinierl** [comment]
Looking at e.g. https://coveralls.io/builds/27683351/source?filename=ocpp-json%2Fsrc%2Fmain%2Fscala%2Fcom%2Fthenewmotion%2Focpp%2Fjson%2Fv1x%2Fv16%2FSerializationV16.scala#L15, it seems like the message translation and serialization suffered. As if the scalacheck tests on these things are not running on Scala 2.13...

**Comment by azolotko** [comment]
@reinierl Great catch! Indeed, after the specs2 update to 4.8.1, the tests don't seem to be running. I'm trying to wrap my head around specs2 internals to understand why this happens, as the issue may apply to other places which don't have coverage report to spot this issue.

**Comment by azolotko** [comment]
@reinierl It turns out that in specs2 4.8+ tests should no longer be wrapped into an `object` and must be `class` instead.

**Comment by azolotko** [comment]
@reinierl coverage is back to the previous level.

**Pull request merged**